backend/src/main/kotlin/org/loculus/backend/config/S3Config.kt

```diff
@@ -0,0 +1,12 @@
+package org.loculus.backend.config
+
+data class S3Config(val enabled: Boolean = false, val bucket: S3BucketConfig?)
+
+data class S3BucketConfig(
+    val endpoint: String,
+    val internalEndpoint: String?,
+    val region: String?,
```

Comment:
region can be null here, but you require it to be non-null here: https://github.com/loculus-project/loculus/pull/3978/files#diff-0d594f2e0976102924e662a789f1b90ad49f0e773d976da2304fc8f0e203472fR125

Reply:
Yes, you're right, that looks wrong! `region` should indeed be optional.

Reply:
(if we make it optional maybe we can not use it for MinIO, which iirc we do atm [which seems weird], but nbd)

Reply:
Sorry, my message is maybe confusing. I was just remembering that when I read the minio config at present it says something like `us-east-1`, which seems weird given we're self-hosting (not in the US!) - so if by changing this we can also avoid that, that's great too!